pyston/locale/cs/LC_MESSAGES/django.po

```diff
@@ -18,32 +18,32 @@ msgstr ""
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=3; plural=(n==1) ? 0 : (n>=2 && n<=4) ? 1 : 2;\n"
 
-#: data_processor.py:89
-msgid "This field is required"
-msgstr "Toto pole je povinné"
-
-#: data_processor.py:109
-msgid "Unsupported file type"
-msgstr "Nepodporovaný typ souboru"
+#: data_processor.py:110
+msgid ""
+"Content type cannot be evaluated from the filename please send it or change "
+"the filename"
+msgstr ""
+"Typ souboru nemůže být vyhodnocen z názvu souboru, prosím pošlete typ "
+"souboru nebo upravte příponu názvu souboru"
```

Comment:
je to přeloženo tak proč se s tím nepočítá v testu, divné že to prochází

Reply:
jj, je to divne, ale prochazi to